include/loops/loops.hpp

```diff
@@ -141,7 +144,7 @@ struct IReg
     IReg();
```

Comment:
I put a comment for a code that has been merged earlier; the comment is about IReg. I suggest to use `int` for index instead of size_t and make NOIDX an enumeration value 

```
struct IReg {
    enum {NOIDX=-1};
    int idx;
}
```
that will make idx twice as compact, that will make each instruction almost twice as compact and it looks better. `static size_t NOIDX` for a special value looks weird

Reply:
Done. I tried not to substitute all size_t counters around code, but dropped off static constant members and made all register idx's integer instead of unsigned.